driver/normalizer/parser.go

```diff
@@ -75,10 +75,26 @@ var ToNoder = &native.ObjectToNoder{
 	// FIXME: test[ast_type=Compare].comparators is a list?? (should be "right")
 }
 
-// ASTParserBuilder creates a parser that transform source code files into *uast.Node.
-func UASTParserBuilder(opts driver.UASTParserOptions) (driver.UASTParser, error) {
+// transUASTParserBuilder creates a parser that transform source code files into *uast.Node
```

Comment:
Function name doesn't match with real one. I'd suggest to remove it, since it's redundant and error-prone